resources/wlcg-scope-tag-migration/WLCGScopeTagList.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * A simple script to home some variables useful for the WLCG Scope tag
```

Comment:
"Home" as in a place for them to live? "Store"?

Reply:
"Store" is better 😄, addressed in https://github.com/GOCDB/gocdb/pull/548/commits/a271f3726521ac7766a45cf2b5d0675787ef2343.